### DISCUSSION TITLE
Missing low fuel warning

### DISCUSSION THREAD

**Discussion by SomeRandomNoobKekeke** [FakeFishGames/Barotrauma#14301] [Bug reports]
### Disclaimers

- [X] I have searched the issue tracker to check if the issue has already been reported.
- [ ] My issue happened while using mods.

### What happened?

If you purchase and switch to new sub without transferring items, low fuel warning will pop-up with fallback values
Language doesn't matter
![20240718001146_1](https://github.com/user-attachments/assets/e4ad8554-35a8-4598-aee3-eaf704fcaa8e)
![20240718001336_1](https://github.com/user-attachments/assets/d5da1fbd-4f32-45c5-ac34-724e4bb32736)

### Reproduction steps

Uncheck "transfer items"
purchase and switch to new sub

### Bug prevalence

Happens every time I play

### Single player or multiplayer?

Single player

### Version

v1.5.9.1 (Summer Update Hotfix 2)

### Which operating system did you encounter this bug on?

Windows

**Comment by Regalis11** [comment]
Fixed in https://github.com/FakeFishGames/Barotrauma-development/commit/a469d829e781c8dffcbe917531c14f31fa75a7f8 (commit in our private repo). The fix will be included in a future update!
